LandSandBoat/server
# [BugFix] Fix Armoury Crate spawning outside BCNM arena

**_I affirm:_**

## What does this pull request do?

Hostile Herbivores BCNM - Arena 2 has an incorrectly configured spawn location for its Armoury Crate.
It was `156.139,-25.788,115.624,6` but should be `-156.139,-25.788,115.624,6`.
This matches all over Arena 2 BCNMs in Horlais Peak

## Steps to test these changes

1. Use 2 characters - not in party.
2. !zone {Horlias Peak}
3. Enter the first into Hostile Herbivores.
4. Enter the second char into Hostile Herbivores.
5. Using the Second Char - win the BCNM
6. Verify that the Armoury Crate Spawns in the BCNM Arena 2.
